client/jetpack-connect/remote-credentials.js

```diff
@@ -0,0 +1,227 @@
+/** @format */
+/**
+ * Component which handle remote credentials for installing Jetpack
+ */
+import React, { Component } from 'react';
+import Gridicon from 'gridicons';
+import page from 'page';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+/**
+ * External dependencies
+ */
+import Button from 'components/button';
+import FormButton from 'components/forms/form-button';
+import FormLabel from 'components/forms/form-label';
+import FormTextInput from 'components/forms/form-text-input';
+import FormattedHeader from 'components/formatted-header';
+import FormPasswordInput from 'components/forms/form-password-input';
+import HelpButton from './help-button';
+import JetpackLogo from 'components/jetpack-logo';
+import LoggedOutForm from 'components/logged-out-form';
+import LoggedOutFormLinks from 'components/logged-out-form/links';
+import LoggedOutFormLinkItem from 'components/logged-out-form/link-item';
+import MainWrapper from './main-wrapper';
+import { addCalypsoEnvQueryArg } from './utils';
+import { externalRedirect } from 'lib/route';
+import { jetpackRemoteInstall } from 'state/jetpack-remote-install/actions';
+import { getJetpackRemoteInstallError, isJetpackRemoteInstallComplete } from 'state/selectors';
+import { getConnectingSite } from 'state/jetpack-connect/selectors';
+import { REMOTE_PATH_AUTH } from './constants';
+
+export class OrgCredentialsForm extends Component {
+	state = {
+		username: '',
+		password: '',
+		submitting: false,
+	};
+
+	getInitialFields() {
+		return {
+			username: '',
+			password: '',
+			submitting: false,
+		};
+	}
+
+	handleSubmit = event => {
+		const { siteToConnect } = this.props;
+		event.preventDefault();
+
+		const url = siteToConnect;
+		const username = this.state.username;
+		const password = this.state.password;
+
+		if ( this.state.submitting ) {
+			return;
+		}
+		this.setState( { submitting: true } );
+
+		this.props.jetpackRemoteInstall( url, username, password );
+	};
+
+	componentDidUpdate() {
+		const { installError, isResponseCompleted, siteToConnect } = this.props;
+
+		if ( isResponseCompleted ) {
+			externalRedirect( addCalypsoEnvQueryArg( siteToConnect + REMOTE_PATH_AUTH ) );
+		}
+		if ( installError ) {
+			//handle errors
+		}
+	}
+
+	getChangeHandler = field => event => {
+		this.setState( { [ field ]: event.target.value } );
+	};
+	getHeaderImage() {
+		return (
+			<div className="jetpack-connect__jetpack-logo">
+				<JetpackLogo full size={ 45 } />
+			</div>
+		);
+	}
+
+	getHeaderText() {
+		const { translate } = this.props;
+
+		return translate( 'Add your website credentials' );
+	}
+
+	getSubHeaderText() {
+		const { translate } = this.props;
+
+		return (
+			<div className="jetpack-connect__install-step">
```

Comment:
This `<div>` gives a warning:

<img width="784" alt="screen shot 2018-02-27 at 12 58 04" src="https://user-images.githubusercontent.com/7767559/36730019-e531120a-1bbd-11e8-8fa2-22322628679c.png">

A `<span>` should be ok if we need the styling.